sound_cloud_backend/user/api/serializers.py

```diff
@@ -0,0 +1,22 @@
+from rest_framework import serializers
+from django.contrib.auth.models import User
+
+class UserSerializer(serializers.ModelSerializer):
+	class Meta:
+		model = User
+		fields = [
+			'username',
+			'first_name',
+			'last_name',
+			'email',
+			'password'
+		]
+
+		extra_kwargs = {'password': {'write_only': True}}
+
+	def create(self, validated_data):
+		password = validated_data.pop('password')
```

Comment:
Make it a normal practice to add a docstring for each class and function. Currently, I think you haven't specified it anywhere.